templates/events/display.html

```diff
@@ -108,19 +142,25 @@ <h5 class="q-mt-none">Buy Ticket</h5>
         dismissMsg()
 
         clearInterval(paymentChecker)
-        setTimeout(function () { }, 10000)
+        setTimeout(function () {}, 10000)
       },
+      nameValidation(val) {
+        let regex = /[`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/g
```

Comment:
- nitpick:
```suggestion
        const regex = /[`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/g
```
- does this expression try to exclude all special characters? isn't it easier (and clearer) to specify the allowed characters in this case?